flow/project.py

```diff
@@ -1821,39 +1959,66 @@ def _fetch_status(self, jobs, err, ignore_errors, status_parallelization='thread
         self._fetch_scheduler_status(jobs, err, ignore_errors)
         # Get status dict for all selected jobs
 
-        def _print_progress(x):
```

Comment:
I think that this is a leftover code hence I removed this. 
@b-butler @csadorf Please confirm.

Reply:
It is.